src/vtkIECTransformLogic.h

```diff
@@ -70,23 +74,27 @@ class vtkGeneralTransform;
   ("s") - PATIENT SUPPORT coordinate system
   ("e") - Table top eccentric rotation coordinate system
   ("t") - Table top coordinate system
-  ("p") - PATIENT coordinate system
+  ("p") - PATIENT coordinate system (LSA)
+ *("dp")- PATIENT coordinate system in LPS (DICOM)
+ *("pi")- Patient image regular grid coordinate system
+*("ras")- PATIENT coordinate system in RAS (3D Slicer)
   ("i") - Imager coordinate system
   ("o") - Focus coordinate system
+   "*"  - Not part of standard IEC coordinate frames
 */
 /*
- IEC Patient to DICOM Patient transformation:
-     Counter clockwise rotation around X-axis, angle = -90 
+ IEC Patient (LSA) to LPS (DICOM) Patient transformation:
+     Counter clockwise rotation around X-axis, angle = -90
 
-                       1 0  0 
+                       1 0  0
      Rotation Matrix = 0 0 -1
```

Comment:
This looks a bit confusing. If I look at https://en.wikipedia.org/wiki/Rotation_matrix#Basic_3D_rotations
and put theta=-90, I would get the transposed matrix of this.
We need to find out why and clarify this in the comments.

Reply:
We found out: this is not a typical rotation matrix "rotating" a vector, but an affine transform matrix expressing the same vector in different coordinate systems (themselves rotated by an angle).
So the affine transform is actually the typical "vector-rotation matrix" with flipped angle sign.

Reply:
To disambiguate, I would suggest:
```suggestion
     TransformMatrix = 0 0 -1
```
Same in the one below.